elementtype.go

```diff
@@ -304,6 +304,10 @@ func (i ElementType) String() string {
 	}
 }
 
+func (i ElementType) Bytes() []byte {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported method ElementType.Bytes should have comment or be unexported